tensorflow_serving/apis/model_management.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+import "tensorflow_serving/config/model_server_config.proto";
+import "tensorflow_serving/util/status.proto";
+
+package tensorflow.serving;
+option cc_enable_arenas = true;
+
+message LoadModelConfigurationRequest {
```

Comment:
nit: let's call it ReloadConfigRequest for consistency with the ServerCore ReloadConfig() method name.

(and similarly ReloadConfigResponse below)